test/basic-querying.test.js

```diff
@@ -6,6 +6,7 @@
 // This test written in mocha+should.js
 'use strict';
 
+/* global getSchema:false, connectorCapabilities:false */
```

Comment:
I agree with both! Should we create an issue to keep track of this work? I am not sure if this problem important enough, there is no point in creating issues that nobody will work for months.